### ISSUE TITLE
Please publish docker image for 3.3.0, 3.3.1, 3.4.0 and 3.4.1

### ISSUE THREAD

**Issue by daniela-waranie** [domibarton/docker-sabnzbd#51]
Release:
- 3.3.0 
- 3.3.1 
- 3.4.0
is available on: https://github.com/domibarton/docker-sabnzbd/tags
Would be cool if you release the docker images on https://hub.docker.com/r/sabnzbd/sabnzbd too.

BTW: 3.4.1 would be cool too.

**Comment by JonasPed** [comment]
And 3.4.2 :-)

**Comment by Archcry** [comment]
Unfortunately docker images are no longer automatically build. @domibarton should probably fix this. @domibarton, could you either fix this or transfer ownership of the dockerhub image to me?

**Comment by foxdan** [comment]
Until this semi-official image gets updated migrating to https://github.com/linuxserver/docker-sabnzbd wasn't overly painful.

Expects different environment variables `PUID/GUID` vs `SABNZBD_UID/SABNZBD_GUID`, `/datadir` becomes `/config` and rename `config.ini` to `sabnzbd.ini`.

**Comment by NimdaBE** [comment]
yes, changing a few things based on what @foxdan posted made the transistion from this repository to the one from linuxserver very easy

**Comment by Mixpower** [comment]
@domibarton Please update or transfer this repository to someone who wants to maintain it, last update has been too long ago!
